automation/src/test/java/com/microsoft/identity/common/test/automation/utility/TestConfigurationHelper.java

```diff
@@ -35,6 +35,11 @@
 
     public static TestConfiguration GetTestConfiguration(TestConfigurationQuery query) {
 
+        //Cloud set
+        if(query.federationProvider == "Cloud") {
+            query.federationProvider = " ";
```

Comment:
Why is this needed ?

Reply:
Your right this wouldn't be needed for this test case. I'll remove until its needed.